onnx/shape_inference/implementation.cc

```diff
@@ -471,6 +487,71 @@ class ShapeInferenceImplBase {
     }
   }
 
+
+ void process(const NodeProto& n, std::unordered_map<std::string, const AttributeProto*> attr_map) {
```

Comment:
Duplicate function? (L396 `void process(NodeProto& n, std::unordered_map<std::string, const AttributeProto*> attr_map)`). If both of them are required, is it possible to make them common in certain way?

Reply:
Good catch. I think it is a duplicate from a merge produced by git. I am mystified why the compiler does not complain, since both have same signature?

Reply:
I see, they differ in `const` ...